### ISSUE TITLE
Grafana dashboards

### ISSUE THREAD

**Issue by StefanoSperetta** [iiacoban42/DelfiTLM#92]
Some general thoughts would be required here. I do not have a clear idea of how to structure them so this is purely a brain dump:

- spit the dashboards by "content" like an overview dashboard and telemetry dashboards
- the overview dashboard would be similar to the top SatNoGS panel  [https://dashboard.satnogs.org/d/O-wbcw1nk/delfi-pq?orgId=1](url) such that it can also be used on a headless screen in the hall
- have telemetry-type based dashboards so we can quickly jump around instead of constant scrolling
- group dashboards per folder (one folder per satellite)
- use panel links from the overview dashboard to link to the other dashboards
- link the overview dashboard to the homepage

Comments, thoughts, suggestions?

**Comment by iiacoban42** [comment]
It's gonna take a bit to get to the sweet spot, you'll probably find it after a few test drives.

**Comment by StefanoSperetta** [comment]
I agree but at least the hierarchy should help us move quicker. The dashboard I care about the most from the public relations point of view is probably the overview: a bit dumb but it should be easy.

With respect to the telemetry ones, I agree, it will take a while while we learn how to operate the system. It might also change over time I guess so for now what you added for Delfi-PQ is perfectly fine.

**Comment by StefanoSperetta** [comment]
Work in progress in the [update_processing](https://github.com/DelfiSpace/DelfiTLM/tree/update_processing) branch...
